src/data/taxonomy-redirects.json

```diff
@@ -361,12 +367,13 @@
     ]
   },
   {
-    "url": "/docs/alerts-applied-intelligence",
+    "url": "/docs/alerts-applied-intelligence/overview",
```

Comment:
@paperclypse, this overview topic is at a parent level and not a child of the first empty node. Aren't we trying for a model where the initial page is always the child of an empty node?